### DISCUSSION TITLE
Filters don't all show off on the web part

### DISCUSSION THREAD

**Discussion by Vblgh-Bruno** [microsoft-search/pnp-modern-search#1799] [Q&A]
Hi everyone,

I come here because I try to apply filters thanks to manage properties. The problem is... They do not appear except for some of them. What can I do to fix this ?

In advance, thank you !
Bruno

![Filter pnp search 2](https://user-images.githubusercontent.com/89794621/152495368-517c3394-53ad-40e0-8ef1-1554681a8bfe.png)
![Filter pnp search](https://user-images.githubusercontent.com/89794621/152495372-cf9aaf8e-edc4-4aff-9663-d65c35e622e0.png)

**Comment by wobba** [comment]
For a property to show it has to have values. Confirm that you have mapped the correct crawled property to the Refinable managed property, and ensure that content is re-indexed after you did the mapping, then check if the items coming from search has those values present.

There are some steps to cover when troubleshooting. I cover most of them in https://www.amazon.com/Troubleshooting-Search-SharePoint-Online-solving-ebook/dp/B082FMGSZY as there are different tools and approaches to figure out the root cause.

**Reply by Vblgh-Bruno** [reply]
Thank You !

And how can I Re-index properly those crawled properties ?

**Reply by wobba** [reply]
Look into the cmdlets `Request-PnPReIndexList` and `Request-PnPReIndexWeb`. If it's all content in a tenant spread on many sites then contact Microsoft support and explain your scenario to have them help on re-indexing all content.
